packages/wallet-sdk/src/models/account.ts

```diff
@@ -0,0 +1,153 @@
+import {
+  ecPairToAddress,
+  getPublicKeyFromPrivate,
+  hashCode,
+  hashSha256Sync,
+  publicKeyToAddress,
+} from '@stacks/encryption';
+import { makeAuthResponse as _makeAuthResponse } from '@stacks/auth';
+import { TransactionVersion, getAddressFromPrivateKey } from '@stacks/transactions';
+import { fromBase58 } from 'bip32';
+import {
+  DEFAULT_PROFILE,
+  fetchAccountProfileUrl,
+  fetchProfileFromUrl,
+  Profile,
+  signAndUploadProfile,
+} from './profile';
+import { ECPair } from 'bitcoinjs-lib';
+import { connectToGaiaHubWithConfig, getHubInfo, makeGaiaAssociationToken } from '../utils';
+import { Buffer } from 'buffer';
```

Comment:
Same here (see previous comment about buffer usage)

Reply:
@zone117x updated these usages to follow the convention.